.ci/pytorch/build.sh

```diff
@@ -355,3 +358,5 @@ if [[ "$BUILD_ENVIRONMENT" != *libtorch* && "$BUILD_ENVIRONMENT" != *bazel* ]];
 fi
 
 print_sccache_stats
+
+sudo chown -R $WORKSPACE_ORIGINAL_OWNER_ID /var/lib/jenkins/workspace
```

Comment:
There is a linting warning here, I'll fix once get greens to avoid loosing the test state